firebase-firestore/firebase-firestore.gradle

```diff
@@ -65,6 +69,7 @@ android {
         versionName version
         multiDexEnabled true
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunnerArguments emulator: "false"
```

Comment:
Did we determine if this line actually does anything?

Reply:
I thought it did, but no, it's just an illusion of control.